lib/c4bb_test_kit/generated/v1.1.0/c4bb_test_suite.rb

```diff
@@ -0,0 +1,111 @@
+require 'inferno/dsl/oauth_credentials'
+require 'smart_app_launch_test_kit'
+require_relative '../../version'
+
+require_relative '../../capability_statement/capability_statement_group'
+
+require_relative 'patient_group'
+require_relative 'coverage_group'
+require_relative 'eob_group'
+require_relative 'eob_inpatient_institutional_group'
+require_relative 'eob_outpatient_institutional_group'
+require_relative 'eob_pharmacy_group'
+require_relative 'eob_professional_non_clinician_group'
+require_relative 'organization_group'
+require_relative 'practitioner_group'
+
+module CarinForBlueButtonTestKit
+  module CARIN4BBV110
+    class C4BBTestKit < Inferno::TestSuite
```

Comment:
This is a test suite, not a test kit.  I assume this was a naming mistake from the existing test kit though.